cmd/k8s-operator/e2e/ingress_test.go

```diff
@@ -89,17 +108,20 @@ func TestIngress(t *testing.T) {
 	}
 
 	var resp *http.Response
-	if err := tstest.WaitFor(time.Second*60, func() error {
+	if err := tstest.WaitFor(60*time.Second, func() error {
 		// TODO(tomhjp): Get the tailnet DNS name from the associated secret instead.
 		// If we are not the first tailnet node with the requested name, we'll get
 		// a -N suffix.
-		resp, err = tsClient.HTTPClient.Get(fmt.Sprintf("http://%s-%s:80", svc.Namespace, svc.Name))
+		req, err := http.NewRequest(httpm.GET, fmt.Sprintf("http://%s-%s:80", svc.Namespace, svc.Name), nil)
```

Comment:
```suggestion
		req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("http://%s-%s:80", svc.Namespace, svc.Name), nil)
```

The stdlib already has constants for HTTP verbs, feel free to ignore if I'm missing something this one provides.

Reply:
We actually have a unit test enforcing that those aren't used 😄  https://github.com/tailscale/tailscale/blob/2d9d869d3dbdf485c9d04276a84435b329d2739f/util/httpm/httpm_test.go#L35

See the `httpm` package docs for details: https://github.com/tailscale/tailscale/blob/2d9d869d3dbdf485c9d04276a84435b329d2739f/util/httpm/httpm.go#L4-L21